test/server.cpp

```diff
@@ -520,15 +521,11 @@ std::string TestContentIn404HtmlResponse::hiddenBookNameInput() const
 
 std::string TestContentIn404HtmlResponse::searchPatternInput() const
 {
-  const std::string searchboxTooltip = isTranslatedVersion()
-                                     ? "Որոնել '" + bookTitle + "'֊ում"
-                                     : "Search '" + bookTitle + "'";
-
-  return R"(          <input autocomplete="off" class="ui-autocomplete-input" id="kiwixsearchbox" name="pattern" type="text" title=")"
-       + searchboxTooltip
-       + R"(" aria-label=")"
-       + searchboxTooltip
-       + R"(">
+  return R"(          <input autocomplete="off" id="kiwixsearchbox" name="pattern" type="text" title="Search ')"
+       + bookTitle
+       + R"('" aria-label="Search ')"
+       + bookTitle
+       + R"('">
```

Comment:
Here we test the translation feature.
We may change a bit the test as we modify the class of the input, be we should still test the
title and aria-label is translated.